executor/executor_test.h

```diff
@@ -5,16 +5,92 @@
 #include <sys/mman.h>
 #include <unistd.h>
 
-#include "nocover.h"
-
 static void os_init(int argc, char** argv, void* data, size_t data_size)
 {
 	void* got = mmap(data, data_size, PROT_READ | PROT_WRITE, MAP_ANON | MAP_PRIVATE | MAP_FIXED, -1, 0);
 	if (data != got)
 		failmsg("mmap of data segment failed", "want %p, got %p", data, got);
+	is_kernel_64_bit = sizeof(void*) == 8;
 }
 
 static intptr_t execute_syscall(const call_t* c, intptr_t a[kMaxArgs])
 {
 	return c->call(a[0], a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8]);
 }
+
+static __thread unsigned long* local_cover_start = NULL;
+static __thread unsigned long* local_cover_end = NULL;
+
+#if EXECUTOR_COVERAGE
```

Comment:
I set it in `buildExecutor()` (pkg/fuzzer/fuzzer_test.go).

Reply:
> Can we make this unconditional and always build test OS with this coverage?

I've dropped the `#if`, now it always compiles the callback.

> But perhaps we can enable coverage as well?

How could it help us in other tests?

Reply:
> How could it help us in other tests?

It will help to ensure that coverage does not lead to crashes, failures, don't corrupt memory, etc, and also remove special code in pkg/fuzzer to build the special version of executor.

Reply:
Added a commit that enables coverage in pkg/runtest tests for Test64Fuzz. I'm not sure it would work with other TestOS targets -- there's an immediate segfault when `-fsanitize=address` is combined with `-fsanitize-coverage=trace-pc` and I think that keeping `-fsanitize=address` for test targets is more important.

Reply:
Is it a segfault or an ASAN report?

Reply:
Posted as a separate issue.